crates/transaction-pool/src/validate/eth.rs

```diff
@@ -185,6 +188,11 @@ impl EthTransactionValidatorBuilder {
         self.eip1559 = eip1559;
         self
     }
+    /// Sets the local transaction propagation toggle
+    pub fn set_local_tx_propagation(mut self, propagate_local_txs: bool) -> Self {
```

Comment:
since true is the default, we can add a

`fn no_local_transaction_propagation(self)`

functions that disables it

Reply:
makes sense, added